backend/framework/qlf/dashboard/bokeh/qacountpix/main.py

```diff
@@ -4,14 +4,16 @@
 from bokeh.models.widgets import PreText, Div
 from bokeh.models import HoverTool, ColumnDataSource, PrintfTickFormatter
 from bokeh.models import LinearColorMapper, ColorBar
+from dashboard.bokeh.qlf_plot import html_table
```

Comment:
This is not actually being used.

```
        tb = html_table(nrng=nrg, wrng=wrg)
        tbinfo = Div(text=tb, width=400)
```

`tbinfo` is not used anywhere

Reply:
ok